unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Debugging/Performance/PerformanceMetricsController.cs

```diff
@@ -135,39 +137,40 @@ private void Report(string encodedSamples)
             float hiccupsTime = tracker.GetHiccupSum();
 
             float totalTime = tracker.GetTotalSeconds();
-            
-            WebInterface.PerformanceReportPayload performanceReportPayload = new WebInterface.PerformanceReportPayload
+
+            var performanceReportRequest = new PerformanceReportRequest
             {
-                samples = encodedSamples,
-                fpsIsCapped = usingFPSCap,
-                hiccupsInThousandFrames = hiccupsInThousandFrames,
-                hiccupsTime = hiccupsTime,
-                totalTime = totalTime,
-                gltfInProgress = gltfloading,
-                gltfFailed = gltffailed,
-                gltfCancelled = gltfcancelled,
-                gltfLoaded = gltfloaded,
-                abInProgress = abloading,
-                abFailed = abfailed,
-                abCancelled = abcancelled,
-                abLoaded = abloaded,
-                gltfTexturesLoaded = gltfTextures,
-                abTexturesLoaded = abTextures,
-                promiseTexturesLoaded = promiseTextures,
-                enqueuedMessages = queuedMessages,
-                processedMessages = processedMessages,
-                playerCount = playerCount,
-                loadRadius = (int)loadRadius,
-                sceneScores = scenesMemoryScore,
-                drawCalls = drawCalls,
-                memoryReserved = totalMemoryReserved,
-                memoryUsage = totalMemoryUsage,
-                totalGCAlloc = totalGCAlloc
+                Samples = encodedSamples,
+                FpsIsCapped = usingFPSCap,
+                HiccupsInThousandFrames = hiccupsInThousandFrames,
+                HiccupsTime = hiccupsTime,
+                TotalTime = totalTime,
+                GltfInProgress = gltfloading,
+                GltfFailed = gltffailed,
+                GltfCancelled = gltfcancelled,
+                GltfLoaded = gltfloaded,
+                AbInProgress = abloading,
+                AbFailed = abfailed,
+                AbCancelled = abcancelled,
+                AbLoaded = abloaded,
+                GltfTexturesLoaded = gltfTextures,
+                AbTexturesLoaded = abTextures,
+                PromiseTexturesLoaded = promiseTextures,
+                EnqueuedMessages = queuedMessages,
+                ProcessedMessages = processedMessages,
+                PlayerCount = playerCount,
+                LoadRadius = (int) loadRadius,
+                SceneScores = { scenesMemoryScore },
+                DrawCalls = (int) drawCalls!,
+                MemoryReserved = (long) totalMemoryReserved!,
+                MemoryUsage = (long) totalMemoryUsage,
+                TotalGcAlloc = (long) totalGCAlloc,
```

Comment:
Wouldn't this case change involve a change in the behavior? Can we have the linter ignore these?

Reply:
In theory, it should not change anything. It's sending with the new RPC instead of JSON.